pr-data.csv

```diff
@@ -1319,7 +1332,7 @@ https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop
 https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core,org.apache.hadoop.mapred.TestLineRecordReader.testUncompressedInputCustomDelimiterPosValue,OD,,,
 https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core,org.apache.hadoop.mapred.TestLineRecordReader.testUncompressedInputDefaultDelimiterPosValue,OD,,,
 https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core,org.apache.hadoop.mapred.TestLineRecordReader.testUncompressedInputWithLargeSplitSize,OD,,,
-https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.mapred.TestLocalMRNotification.testMR,ID,,,
+https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.mapred.TestLocalMRNotification.testMR,ID,DeveloperFixed,,619e490333fa89601fd476dedac6d16610e9a52a
```

Comment:
How did you find the test was fixed in this commit? I checked the [commit (https://github.com/apache/hadoop/commit/619e490333fa89601fd476dedac6d16610e9a52a). It only changed the markdown

Reply:
Oh no, I simply run the script in `auto-check-fix-commit`.

Reply:
> Oh no, I simply run the script in `auto-check-fix-commit`.

Please double check, it's unlikely that this commit fixes this test

Reply:
Ok, thanks!